### PR TITLE
Documentation update

### LINKED ISSUE #90: Documentation Update including Docker Setup
There is an update for a Docker setup which requires some changes in README.md to clarify how to use this on localhost.

### PR DESCRIPTION
**Pull request by ghost**
Adding information about installing docker and docker compose as well as updating the initial instructions for getting up and running to README.md and CONTRIUBTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,13 +45,18 @@ If you're a designer and would like to contribute, feel free to reach out to us!
 The app is built with the following code dependencies:
 
 1. [NextJS](https://github.com/vercel/next.js)
+2. [Express](https://github.com/expressjs/express)
+3. [PostgreSQL](https://github.com/postgres/postgres)
```
Comment (ZumDeWald) [inline README.md:49]: Since this is open, could we add Styled Components? Seeing how we are using it on the Front-End.

**Review by ZumDeWald — COMMENTED** [review]
Just thought I'd ask to add Styled Components to the dependencies since you have added others. The rest looks good for now!

**Comment by silentkdev** [comment]
@ZumDeWald added the requested change

**Review by ZumDeWald — APPROVED** [review]

**Review by cMikolai — APPROVED** [review]
Yeah! Well done, thank you!

**Pull request merged**